system/system.go

```diff
@@ -143,12 +149,25 @@ func isRpm() bool {
 	}
 
 	// See if it has only one of the package managers
-	if hasCommand("rpm") && !hasCommand("dpkg") {
+	if hasCommand("rpm") && !hasCommand("dpkg") && !hasCommand("apk") {
 		return true
 	}
 	return false
 }
 
+func isAlpine() bool {
+	if _, err := os.Stat("/etc/alpine-release"); err == nil {
+		return true
+	}
+
+	// See if it has only one of the package managers
+	if hasCommand("dpkg") && !hasCommand("rpm") && hasCommand("apk") {
```

Comment:
derp ... it's still early in Seattle. pushed fix and README changes

On Tue, Jan 19, 2016 at 8:16 AM, Ahmed Elsabbahy <notifications@github.com>
wrote:

> In system/system.go
> <https://github.com/aelsabbahy/goss/pull/35#discussion_r50133315>:
>
> >  		return true
> >  	}
> >  	return false
> >  }
> >
> > +func isAlpine() bool {
> > +	if _, err := os.Stat("/etc/alpine-release"); err == nil {
> > +		return true
> > +	}
> > +
> > +	// See if it has only one of the package managers
> > +	if hasCommand("dpkg") && !hasCommand("rpm") && hasCommand("apk") {
>
> Does alpine use dpkg? If not, the first conditional should be a !.
>
> Also can you update the readme limitations section, otherwise looks good.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aelsabbahy/goss/pull/35/files#r50133315>.
>